ingestion-lambda/src/handler.ts

```diff
@@ -95,27 +99,13 @@ export const decodeBodyTextContent = (
 		})
 		.replace(/\n/g, '<br />');
 
-export const safeBodyParse = (body: string): IngestorInputBody => {
+export const safeJsonParse = (body: string): Record<string, unknown> => {
```

Comment:
this needed splitting into two steps, otherwise the "fall back" step which cleans the curly quotes and/or fixes control chars will not have the preprocessed keywords or body content